client/_app.js

```diff
@@ -1,7 +1,11 @@
 import React from "react";
-import ApolloClient from "apollo-boost";
-import { ApolloProvider } from "react-apollo";
-import { AppProvider as PolarisProvider } from "@shopify/polaris";
+import {
+  ApolloClient,
+  ApolloProvider,
+  InMemoryCache,
+  HttpLink,
+} from "@apollo/client";
+import { AppProvider as PolarisProvider, Page } from "@shopify/polaris";
```

Comment:
Do we need the `Page` import here? Doesn't seem like we're using it.

Reply:
ah 🤦‍♀️ oops didnt mean to leave that in there